src/js/base/module/Placeholder.js

```diff
@@ -5,6 +5,11 @@ export default class Placeholder {
 
     this.$editingArea = context.layoutInfo.editingArea;
     this.options = context.options;
+    // apply original placeholder
+    if (this.options.inheritPlaceholder) {
```

Comment:
why don't `this.options.placeholder === true` to tell summernote load placeholder content from attr?

Reply:
OK. Explicit is better than implicit.